src/test/java/org/icgc/argo/workflow_management/util/WorkflowTest.java

```diff
@@ -1,57 +0,0 @@
-/*
```

Comment:
Deleted cause this file wasn't testing anything. It seems to be more of a local test runner but I don't think it works cause in line 54 result is Mono<> but its not subscribed to so nothing happens.